File: Search Key-1/src/SearchKey.java
import java.util.Scanner;

public class SearchKey {

	public static void main(String[] args) {
		int i;
		System.out.println("Enter array limit : ");
		Scanner sc=new Scanner(System.in);
		int limit=sc.nextInt();
		
		int [] a=new int[100];
		System.out.println("Enter values : ");
		for(i=0;i<limit;i++) {
			a[i]=sc.nextInt();
		}
		int flag=0;
		System.out.println("Enter search key : ");
	    int searchkey=sc.nextInt();
	    for(i=0;i<limit;i++) {
			if(searchkey==a[i]) {
			   flag=1;
				break;
			}
		} 
	    if(flag==1) {
	    	System.out.println("Entered value in "+(i+1)+" position");
	    }else {
	    	System.out.println("Value not found");
	    }
	}
}

File: Search Key/src/Search Key.c
/*
 ============================================================================
 Name        : Search.c
 Author      : 
 Version     :
 Copyright   : Your copyright notice
 Description : Hello World in C, Ansi-style
 ============================================================================
 */

#include <stdio.h>
#include <stdlib.h>

int main(void) {
	int i,a[5],searchkey,flag=0;
	setbuf(stdout,NULL);
	printf("Enter values : ");

	for(i=0;i<5;i++){
		scanf("%d",&a[i]);
	}
	printf("Please enter search key  : ");
	scanf("%d",&searchkey);
	for(i=0;i<5;i++){
			if(searchkey==a[i]){
				flag=1;
				break;
			}
		}
	if(flag==1){
		printf("It is in %d position",i+1);
	}else {
		printf("value not found");
	}
	return EXIT_SUCCESS;
}
